polaris-react/src/components/Badge/components/Pip/Pip.scss

```diff
@@ -0,0 +1,69 @@
+.Pip {
+  --pc-pip-size: var(--p-space-2);
+  --pc-pip-color: var(--p-icon);
+  display: inline-block;
+  color: var(--pc-pip-color);
+  height: var(--pc-pip-size);
+  width: var(--pc-pip-size);
+  border: var(--p-border-width-2) solid var(--pc-pip-color);
+  border-radius: var(--p-border-radius-half);
+  flex-shrink: 0;
+}
+
+.statusSuccess {
+  --pc-pip-color: var(--p-icon-success);
+}
+
+.statusInfo {
+  --pc-pip-color: var(--p-icon-highlight);
```

Comment:
This is is the color for `statusNew`, `statusInfo` should've been `--p-icon` not `--p-icon-highlight`. Will cut a patch release with a fix.